2_process.R

```diff
@@ -196,36 +268,43 @@ p2b_targets_list <- list(
   # the network due to odd PRMS-SNTemp values and so Barclay et al. (In Prep, 2023)
   # predicts temperature on 455 reaches in the DRB. 
   tar_target(
-    p2b_static_inputs_nhm_combined,
-    p2b_static_inputs_nhm_formatted %>%
-      left_join(y = p2a_static_inputs_prms, by = "seg_id_nat") %>%
-      left_join(y = p2b_soller_coarse_sediment_reaches_nhm  %>%
+    p2_static_inputs_nhm_combined,
+    p2_static_inputs_nhm_formatted_for_merge %>%
+      left_join(y = p2_static_inputs_prms, by = "seg_id_nat") %>%
+      left_join(y = p2_soller_coarse_sediment_reaches_nhm  %>%
                   sf::st_drop_geometry() %>% 
                   select(seg_id_nat, cs_area_proportion) %>% 
                   rename(prop_reach_w_coarse_sediment = cs_area_proportion),
                 by = "seg_id_nat") %>%
-      left_join(y = p2b_depth_to_bedrock_reaches_along_nhm %>%
+      left_join(y = p2_depth_to_bedrock_reaches_along_nhm %>%
                   sf::st_drop_geometry() %>%
                   rename(dtb_weighted_mean_reach = dtb_weighted_mean),
                 by = "seg_id_nat") %>%
-      left_join(y = p2b_depth_to_bedrock_catchments_along_nhm_dissolved %>%
+      left_join(y = p2_depth_to_bedrock_catchments_along_nhm_dissolved %>%
                   sf::st_drop_geometry() %>%
                   rename(dtb_weighted_mean_catchment = dtb_weighted_mean),
                 by = "seg_id_nat") %>%
-      left_join(y = rename(p2b_confinement_mcmanamay_filled, flag_gaps_mcmanamay = flag_gaps), 
+      left_join(y = rename(p2_confinement_mcmanamay_filled, flag_gaps_mcmanamay = flag_gaps), 
                 by = "seg_id_nat") %>%
-      left_join(y = rename(p2b_confinement_facet_filled, flag_gaps_facet = flag_gaps), 
+      left_join(y = rename(p2_confinement_facet_filled, flag_gaps_facet = flag_gaps), 
                 by = "seg_id_nat") %>%
-      left_join(y = p2b_nhdv2_attr, by = "seg_id_nat") %>%
+      left_join(y = p2_nhdv2_attr, by = "seg_id_nat") %>%
       mutate(seg_id_nat = as.integer(seg_id_nat)) %>%
       left_join(y = select(p1_modflow_params, -COMID), by = "seg_id_nat") %>%
       left_join(y = select(p1_modflow_discharge, -COMID), by = "seg_id_nat")
   ),
   
+  # removing attributes that are not needed for final output dataset for model and model archive 
+  tar_target(p2_static_inputs_nhm_combined_model_archive, 
+             p2_static_inputs_nhm_combined |> select(!all_of(static_inputs_nhm_to_remove))
+  ),
+  
+  
   # Save a feather file that contains the formatted NHM-scale attributes
   tar_target(
-    p2b_static_inputs_nhm_formatted_feather,
-    write_feather(p2b_static_inputs_nhm_combined, sprintf("2b_process_nhm_groundwater/out/nhm_attributes_%s.feather", format(Sys.Date(), "%Y%m%d"))),
+    p2_static_inputs_nhm_formatted_feather,
+    write_feather(p2_static_inputs_nhm_combined_model_archive,
+                  sprintf("2b_process_nhm_groundwater/out/nhm_attributes_%s.feather", format(Sys.Date(), "%Y%m%d"))),
```

Comment:
Does this suggestion seem right to you, given the changes in this PR?

```suggestion
                  sprintf("2_process_nhm_groundwater/out/nhm_attributes_%s.feather", format(Sys.Date(), "%Y%m%d"))),
```